integration_tests/on_demand_service_broker/binding_test.go

```diff
@@ -48,19 +50,31 @@ var _ = Describe("binding service instances", func() {
 		}
 	)
 
+	BeforeEach(func() {
+		cfClient = "cf"
+	})
+
 	JustBeforeEach(func() {
+
 		boshUAA = mockuaa.NewClientCredentialsServer(boshClientID, boshClientSecret, "bosh uaa token")
 		boshDirector = mockbosh.NewWithUAA(boshUAA.URL)
 		cfUAA = mockuaa.NewClientCredentialsServer(cfUaaClientID, cfUaaClientSecret, "CF UAA token")
 		cfAPI = mockcfapi.New()
 
-		var brokerConfig config.Config
 		brokerConfig = defaultBrokerConfig(boshDirector.URL, boshUAA.URL, cfAPI.URL, cfUAA.URL)
 
-		runningBroker = startBrokerWithPassingStartupChecks(brokerConfig, cfAPI, boshDirector)
+		switch cfClient {
+		case "dummy":
```

Comment:
In reference to an above comment, would it be possible to replace "dummy" with "NoopCFClient"?